### DISCUSSION TITLE
Show full fraction in cases function

### DISCUSSION THREAD

**Discussion by Silzinc** [typst/typst#5584] [Q&A] [answered]
Hello! The fractions go to inline mode if I write them in a `cases` instance.
With `cases`:
![image](https://github.com/user-attachments/assets/f228c216-7463-45cc-acca-3562783e58c0)
Without:
![image](https://github.com/user-attachments/assets/bc0e244a-2cbc-4784-b334-fe28f3669563)

Generally speaking, I'd like to know how to replicate LaTeX's `dfrac` macro. I saw [this thread](https://github.com/typst/typst/discussions/1116) but none of the solutions seemed to work for me. Thank you in advance for helping out!

**Comment by Enivex** [comment, marked as answer]
The display() function is what you're looking for. That thread is _very_ old

**Reply by Silzinc** [reply]
Thank you! I should blame myself for not looking hard enough, though I'd also like to blame my search engine and github's thread search for not giving me the answer right away.
